core/wiki/macros/translink.tid

```diff
@@ -3,25 +3,13 @@ tags: $:/tags/Macro
 
 \define translink(title,mode:"block")
 \whitespace trim
-<div class={{{ [[tc-translink]] [[$title$]is[missing]then[tc-translink-inline]] [[$mode$]match[block]then[tc-translink-block]] [[$mode$]match[inline]then[tc-translink-inline]] +[first[2]] +[join[ ]] }}}>
+<div class="pre-border" style="padding: 0 1em;">
 <$link to="""$title$""">
-<h1 style="font-weight: bold; font-size: 1.2em;"><$text text="""$title$"""/></h1>
+<h1 class="tc-big-bold"><$text text="""$title$"""/></h1>
 </$link>
 <$transclude tiddler="""$title$""" mode="$mode$">
 "<$text text="""$title$"""/>" is missing
 </$transclude>
+<$list filter="[<__title__>is[missing]] ~[<__mode__>match[inline]]"><p></p></$list>
```

Comment:
`<$list filter="[<__title__>is[missing]] ~[<__mode__>match[inline]]"><p></p></$list>` .. what is the `<p></p>` good for. It will always be empty. So **if it is used to introduce vertical spacing that's the wrong way to do it**. There should a class assigend to the right elements.

Reply:
I have just changed it, the spacing is achieved the proper way now.